notebooks/official/workbench/spark/spark_ml.ipynb

```diff
@@ -0,0 +1,1623 @@
+{
```

Comment:
<p>Is this still necessary in this sample? I know it was for the other notebook sample.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/838/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>When the dataset is loaded from <code>spark-bigquery-connector</code> , the Citi Bike dataset is filtered that only contains trips in 2016. I initially tried to fetch every rows and filtered out using the spark command, but filtering in cluster took more time than fetching filtered data from BQ. To enable to query data in <code>spark-bigquery-connector</code> , <code>materializationDataset</code>  must be set to a dataset. Please check <a href="https://github.com/GoogleCloudDataproc/spark-bigquery-connector#reading-data-from-a-bigquery-query" target="_blank">this link</a>. </p>

Reply:
<p>This is not applicable since we don't use BQ anymore in this project.</p>